[2022-02-24 21:55:20,359] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: meu_dag.criar_arquivo manual__2022-02-24T21:55:19.813674+00:00 [queued]>
[2022-02-24 21:55:20,374] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: meu_dag.criar_arquivo manual__2022-02-24T21:55:19.813674+00:00 [queued]>
[2022-02-24 21:55:20,375] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-24 21:55:20,375] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-24 21:55:20,376] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-24 21:55:20,391] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): criar_arquivo> on 2022-02-24 21:55:19.813674+00:00
[2022-02-24 21:55:20,398] {standard_task_runner.py:52} INFO - Started process 790 to run task
[2022-02-24 21:55:20,409] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'meu_dag', 'criar_arquivo', 'manual__2022-02-24T21:55:19.813674+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/dag_xcom_test.py', '--cfg-path', '/tmp/tmpfphpu1tp', '--error-file', '/tmp/tmpu3a1glid']
[2022-02-24 21:55:20,410] {standard_task_runner.py:77} INFO - Job 5: Subtask criar_arquivo
[2022-02-24 21:55:20,486] {logging_mixin.py:109} INFO - Running <TaskInstance: meu_dag.criar_arquivo manual__2022-02-24T21:55:19.813674+00:00 [running]> on host d86b6291a432
[2022-02-24 21:55:20,551] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=meu_dag
AIRFLOW_CTX_TASK_ID=criar_arquivo
AIRFLOW_CTX_EXECUTION_DATE=2022-02-24T21:55:19.813674+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-24T21:55:19.813674+00:00
[2022-02-24 21:55:20,552] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-24 21:55:20,554] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'touch ~/meu_arquivo_01.txt']
[2022-02-24 21:55:20,566] {subprocess.py:85} INFO - Output:
[2022-02-24 21:55:20,571] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-24 21:55:20,599] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=meu_dag, task_id=criar_arquivo, execution_date=20220224T215519, start_date=20220224T215520, end_date=20220224T215520
[2022-02-24 21:55:20,656] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-24 21:55:20,694] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
